src/server/server_family.cc

```diff
@@ -61,6 +61,8 @@ ABSL_FLAG(string, save_schedule, "",
           "glob spec for the UTC time to save a snapshot which matches HH:MM 24h time");
 ABSL_FLAG(bool, df_snapshot_format, true,
           "if true, save in dragonfly-specific snapshotting format");
+ABSL_FLAG(int, epoll_queue_threads, 0,
+          "fiber queue thread size when running in epoll mode, default is hardware concurrent threads supported");
```

Comment:
`thread size for file workers when running in epoll mode, default is hardware concurrent threads`